backend/bitboxbase/bitboxbase.go

```diff
@@ -232,22 +243,22 @@ func (base *BitBoxBase) VerificationProgress() (rpcmessages.VerificationProgress
 	return base.rpcClient.GetVerificationProgress()
 }
 
-// SyncOption returns true if the chosen sync option was executed successfully
-func (base *BitBoxBase) SyncOption(option string) (bool, error) {
+// SyncOptionStatus returns true if the chosen sync option was executed successfully
+func (base *BitBoxBase) SyncOptionStatus(option SyncOption) (bool, error) {
```

Comment:
imho this is not a good name, as it is not fetching a status, but performing the sync action?

What about `SyncWithOption`?

Reply:
Nice, sounds better